src/storage/mutate/AddEdgesProcessor.cpp

```diff
@@ -13,27 +13,163 @@ namespace nebula {
 namespace storage {
 
 void AddEdgesProcessor::process(const cpp2::AddEdgesRequest& req) {
-    auto spaceId = req.get_space_id();
+    spaceId_ = req.get_space_id();
     auto version =
         std::numeric_limits<int64_t>::max() - time::WallClock::fastNowInMicroSec();
     // Switch version to big-endian, make sure the key is in ordered.
     version = folly::Endian::big(version);
 
     callingNum_ = req.parts.size();
+    auto iRet = schemaMan_->getEdgeIndexes(spaceId_);
+    if (iRet.ok()) {
+        for (auto& index : iRet.value()) {
+            indexes_.emplace_back(index);
+        }
+    }
```

Comment:
What if !iRet.ok(),  return error directly.

Reply:
This schemaMan_ is going to be replaced by indexMan_ .
Currently, return` !iRet.ok()` means that the index does not exist